cdrom/aulas/static/aulas/css/main.css

```
@@ -1,3 +1,10 @@
+.body{
```

```suggestion
body{
```
Usar el objeto body, no necesitamos una clase